Code/MessageBoxHandler.cs

```diff
@@ -178,6 +178,9 @@ private void SetIconTexture()
 				{
 					int level = ModCore.Utility.GetPlayer().GetStateVariable("melee");
 
+					if (level > 1)
```

Comment:
Will this not just immediately bleed into line 184 and set the melee level to `level` instead of `Melee2`? What's the point of this?

Reply:
lol that is a good point, and is why it helps to have someone else look at the code :mjau: Will fix